jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -112,7 +172,14 @@ public String getLoggerLevel(String loggerName) {
 	public List<String> getLoggerNames() {
 		// By default, return an empty list to caller
 		List<String> result = new ArrayList<>();
-		Enumeration<String> enumeration = LogManager.getLogManager().getLoggerNames();
+		Enumeration<String> enumeration;
```

Comment:
Can you change the type to `Enumeration<?>` to avoid 'Unchecked cast' warning?
(a cast will be needed to the value returned by `enumeration.nextElement()`).